site/src/pages/WorkspacePage/WorkspacePage.tsx

```diff
@@ -34,46 +35,31 @@ export const WorkspacePage: FC = () => {
   const { username, workspace: workspaceName } = useParams() as {
     username: string
     workspace: string
+    template: string
```

Comment:
Why do we have to cast here? I am not seeing a TS error if I remove `as`.

Reply:
we don't need this, good catch